stdlib/float.mli

```diff
@@ -217,14 +278,37 @@ external floor : float -> float = "caml_floor_float" "floor"
     equal to [f].
     The result is returned as a float. *)
 
-external copysign : float -> float -> float
+external next_after : float -> float -> float
```

Comment:
Is the function actually useful to expose, considering succ/pred?  Do we expect cases where the caller actually need the semantics of next_after and cannot use succ/pred instead?

Reply:
> useful to expose, considering succ/pred

Absolutely.  For example `next_after x 0.` will “truncate” `x` regardless of its sign;  `next_after x y = y` says whether the interval [`x`, `y`] is tight.

Reply:
Ok, thanks!